src/viur/core/tasks.py

```diff
@@ -655,7 +655,11 @@ def mkDecorator(fn):
                                f"Please rename {fn.__name__!r}")
         if cronName not in _periodicTasks:
             _periodicTasks[cronName] = {}
-        _periodicTasks[cronName][fn] = interval
+        if isinstance(interval, datetime.timedelta):
+            # convert seconds to minutes
+            _periodicTasks[cronName][fn] = int(interval.total_seconds() / 60)
+        else:
+            _periodicTasks[cronName][fn] = interval
```

Comment:
Should we enforce int here?
```suggestion
            _periodicTasks[cronName][fn] = int(interval)
```